src/main/kotlin/no/nav/familie/ef/sak/behandling/Saksbehandling.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.familie.ef.sak.behandling
+
+import no.nav.familie.ef.sak.behandling.domain.BehandlingResultat
+import no.nav.familie.ef.sak.behandling.domain.BehandlingStatus
+import no.nav.familie.ef.sak.behandling.domain.BehandlingType
+import no.nav.familie.ef.sak.behandling.dto.HenlagtÅrsak
+import no.nav.familie.ef.sak.behandlingsflyt.steg.StegType
+import no.nav.familie.ef.sak.fagsak.domain.Stønadstype
+import no.nav.familie.kontrakter.ef.felles.BehandlingÅrsak
+import org.springframework.data.relational.core.mapping.Column
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
```

Comment:
Jeg har ikke hørt diskusjonen rundt hvilke felter som er interessante her, men ville `Vedtak.resultattype` være et interessant felt i dette aggregatet?

Reply:
Vet ikke om det trengs i dette aggregat, men har ikke sett noe på det
Mulig man kan ta det i en ny PR hvis det er behov for det

Reply:
Enig, tar det i en PR når/om behovet dukker opp.